config/supported_ssg.yaml

```diff
@@ -82,6 +82,7 @@ supported:
   RHEL-7.2:
     - package: scap-security-guide-0.1.25-3.el7
       version: 0.1.25
+      real_version: 0.9
```

Comment:
Ok, now revisiting this, I think this is a fine way to try and tackle this issue; you can disregard my earlier comments. One thing we should also change is to treat reports coming in against version 0.9 as 0.1.25 as well. I suspect we will only ever have this one version that's problematic, and luckily `0.9` doesn't conflict with any other SSG versions.

Reply:
Wait, isn't my code already doing that? the `save_missing_supported_benchmark` is IN the report parser :thinking:

Reply:
Hmm... maybe you're right! I guess that's the only place the SSG version is even referenced. :laughing: Ok, I'm going to test this so we can get it merged :+1: